kubernetes_deploy/dev-agfs/deployment.yaml

```diff
@@ -0,0 +1,231 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: claim-for-crown-court-defence
+  namespace: cccd-dev-agfs
+spec:
+  replicas: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 100%
+  selector:
+    matchLabels:
+      app: cccd
+  template:
+    metadata:
+      labels:
+        app: cccd
+    spec:
+      containers:
+        - name: cccd-app
+          imagePullPolicy: Always
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/laa-get-paid/cccd:set-me
+          ports:
+            - containerPort: 3000
+          readinessProbe:
+            httpGet:
+              path: /ping.json
+              port: 3000
+              httpHeaders:
+                - name: X-Forwarded-Proto
+                  value: https
+                - name: X-Forwarded-Ssl
+                  value: "on"
+            initialDelaySeconds: 15
+            periodSeconds: 10
+          livenessProbe:
+            httpGet:
+              path: /ping.json
+              port: 3000
+              httpHeaders:
+                - name: X-Forwarded-Proto
+                  value: https
+                - name: X-Forwarded-Ssl
+                  value: "on"
+            initialDelaySeconds: 30
+            periodSeconds: 10
+          env:
+            - name: ENV
+              value: 'dev'
```

Comment:
as [this comment](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3273/files#r411298689) but dev-agfs.

if impact is problematic then i think we can get away with calling these envs `dev`